README.md

```diff
@@ -1,2 +1,67 @@
-# ToDo_Peer_Review
-This is a code review  project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+This is a web app to orgaise your tasks in a digital shelf. This app is built with  Javascript.
+![image](https://user-images.githubusercontent.com/37586807/184165479-386933f2-0bee-420b-bc67-9a0826abfbad.png)
+
+
+
+## Built With
+
+- Javascript, HTML
+- weblint, eslint, hint
+- webpack
+
+
+## Getting Started
+git clone https://github.com/GloireIrenge/ToDoList.git
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+The basic requirements for building the executable are:
+
+- A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+- VSCode or any other equivalent code editor :atom:
+- Node Package Manager (For running, debuging, or installing dependancies to be used)
+
+### Install
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
```

Comment:
To run your program, other developers need to install the `stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` package and any other dependencies that your program requires. 

You can simplify this process by running `npm install` in the project directory. This will install all of the dependencies that your program requires, making it easier for other developers to get started.